topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -343,10 +377,63 @@ For the next step we need additional data tracks. Please load `dm3_genes.bed`, `
 
 The resulting image should look like this one:
 ![TAD plot](../../images/plotTADs.png)
+
+# Loop detection
+
+In Hi-C data, the term `loop` refers to a 3D structure which represent enhancer-promoter, gene, architectural or polycomb-mediated interactions. These interactions have the characteristics to be enriched in a single region compared to the local background. These loops are also so called long-range interactions with an expected maximum distance of 2 MB (see [Rao et al. 2014](https://doi.org/10.1016/j.cell.2014.11.021).
+
+![Loops visualization](../../images/loops_bonev_cavalli.png)
+
+To compute loops, we have to import a new data set from the shared library to our history: `GM12878-MboI-allreps-filtered.10kb.cool` or download it via the <a href="ftp://cooler.csail.mit.edu/coolers/hg19/Rao2014-GM12878-MboI-allreps-filtered.10kb.cool">FTP cool files server</a>. (FTP links seems to be blocked, therefore the full URL: `ftp://cooler.csail.mit.edu/coolers/hg19/Rao2014-GM12878-MboI-allreps-filtered.10kb.cool`)
+
+This dataset is from the human cell GM12878, mapped to hg19 and of 10 kb resolution. We use a new file because to detect loop structures the read coverage is required to be in the hunderts of million; this was not the case for the previous used drosophila dataset.
+
+> ### {% icon hands_on %} Hands-on: Matrix information 
+>
+> 1. **hicInfo** {% icon tool %}: Run hicInfo adjusting the parameters:
+>    - "Select" `Multiple datasets`
+>    - "Matrix to compute on" to `corrected contact matrix dm3 large` and `GM12878-MboI-allreps-filtered.10kb.cool`
+{: .hands_on}
+
+We now investigate the result of hicInfo and see that the new importet file is having 1.2 billion non-zero elements, while the drosophila Hi-C interaction matrix has around 12 million non-zero elements.
+
+> ### {% icon hands_on %} Hands-on: Computing loops
+>
+> 1. **hicDetectLoops** {% icon tool %}: Run hicDetectLoops adjusting the parameters:
+>    - "Matrix to compute on" to `GM12878-MboI-allreps-filtered.10kb.cool`
+>    - "Peak width" to `6`
+>    - "Window size" to `10`
+>    - "P-value preselection" to `0.01`
+>    - "P-value" to `0.01`
+>    - "Chromosomes to include" to `chr1`
+>
+{: .hands_on}
+
+
+The detection of the loops is based on a preselection of interactions, per interaction a p-value given a continuous negative binomial distribution over all interactions of a relative distance is computed. In a second step, the selected peak candidate is compared against its background using a Wilcoxon rank-sum test.

```

Comment:
`the pre-selection of interactions ..., p-value is computed per interaction given a continuous negative ...`